Formula/sile.rb

```diff
@@ -34,6 +63,17 @@ def install
                           "--prefix=#{prefix}"
     system "make"
     system "make", "install"
+    # There's a handful of broken dylibs reported to old `icu4c` versions
+    # and such. This seems to be resolved in HEAD. Not sure what's going on
```

Comment:
No, not yet. It doesn't help that between the last release & the current master there's been a _ton_ of commits including some major rewriting & I haven't fancied bisecting the whole party between the last release & now yet 😢.

There's still `lua-zlib`'s ridiculous behaviour to report upstream to `lua-zlib` as well.